xpublish_wms/getmap.py

```diff
@@ -0,0 +1,333 @@
+import io
+import time
+from datetime import datetime
+from typing import List
+
+import cachey
+import cf_xarray  # noqa
+import xarray as xr
+import pandas as pd
+import numpy as np
+from loguru import logger
```

Comment:
I use loguru instead of standard logger, because so much easy to setup

But I don't mind to switch back to `logging`

Reply:
I would rather not deal with an additional logging package for now, keep it slim. Thanks!